src/components/buttonMore/buttonmore.js

```diff
@@ -0,0 +1,38 @@
+import template from "./buttonmore.html";
```

Comment:
please be consistent in naming files -- we use kebab-case for file naming -- so it should be button-more/button-more.html